src/libraries/System.Net.WebProxy/src/System/Net/WebProxy.cs

```diff
@@ -219,7 +224,10 @@ private sealed class ChangeTrackingArrayList : ArrayList
 
             public ChangeTrackingArrayList(ICollection c) : base(c) { }
 
-            public bool IsChanged { get; set; }
+            // While this type isn't intended to mutated concurrently with reads, non-concurrent updates
+            // to the list might result in lazy initialization, and it's possible concurrent requests could race
+            // to trigger that initialization.
```

Comment:
```suggestion
            // While this type isn't intended to be mutated concurrently with reads, non-concurrent updates
            // to the list might result in lazy initialization, and it's possible concurrent reads could race
            // to trigger that initialization.
```

Reply:
Non-blocking feedback - just minor comment language changes.

Reply:
FWIW, the "requests" was intended to be referring to HTTP requests resulting in calls to IWebProxy.GetProxy.

Reply:
Ah, thanks for clarification -- updated above with "HTTP requests" instead.